pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarConnectorConfig.java

```diff
@@ -191,6 +195,26 @@ public PulsarConnectorConfig setOffloaderProperties(String offloaderProperties)
         return this;
     }
 
+    public String getRewriteNamespaceDelimiter() {
+        return rewriteNamespaceDelimiter;
+    }
+
+    @Config("pulsar.rewrite-namespace-delimiter")
+    public PulsarConnectorConfig setRewriteNamespaceDelimiter(String rewriteNamespaceDelimiter) {
+        this.rewriteNamespaceDelimiter = rewriteNamespaceDelimiter;
```

Comment:
@codelipenghui can you write a simple check to make sure the delimiter is not an allowed character for tenant and namespace names.